wandb/apis/public/artifacts.py

```diff
@@ -303,6 +304,8 @@ def __init__(
         self._tags = [a["node"]["name"] for a in self._attrs["tags"]["edges"]]
         self._saved_tags = copy(self._tags)
         self.organization = organization
+        if is_sequence is not None:
```

Comment:
I don't think this param should override the assignment that comes through `load()`. maybe we should move it above line 299? wdyt?

Reply:
Or we could ignore this parameter if `attr is None`?